handler/handler.go

```diff
@@ -28,8 +28,8 @@ func (HandlerServer) Handle(ctx context.Context, call Handler_handle) error {
 	// TODO: could get fancy here and implement more fancy logic on entire request
 	req, _ := call.Args().Request()
```

Comment:
You'll want to check this error and return it from the handler if it's not nil.

It's a very bad idea to ignore errors in Go.